pkg/syncer/metadatasyncer.go

```diff
@@ -41,6 +44,29 @@ func NewInformer() *MetadataSyncInformer {
 	return &MetadataSyncInformer{}
 }
 
+// getFullSyncIntervalInMin return the FullSyncInterval
+// If enviroment variable X_CSI_FULL_SYNC_INTERVAL_MINUTES is set and valid,
+// return the interval value read from enviroment variable
+// otherwise, use the default value 30 minutes
+func getFullSyncIntervalInMin() int {
+	fullSyncIntervalInMin := defaultFullSyncIntervalInMin
+	if v := os.Getenv("X_CSI_FULL_SYNC_INTERVAL_MINUTES"); v != "" {
```

Comment:
I feel like this should be defined as a constant somewhere. Then it wouldn't also have to be typed out in each of the `klog` lines below.

Also, along the same thing thinking as https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/52#discussion_r319592404, I don't see any reason for this to be prefixed with `X_CSI`.

Reply:
Travis, thanks for the review. I have addressed your comment, please review it.